src/main/java/collab/Main.java

```diff
@@ -1,9 +1,29 @@
 package collab;
 
+import java.util.List;
+
 public class Main {
 
     public static void main(String[] args) {
-        System.out.println("Hello");
-
+        if (args.length !=2) {
+            System.out.println("Usage: Main.exe input.txt output.txt");
+        }
+        String inFilePath = args[0];
+        //System.out.println("in: " + inFilePath);
+        String outFilePath = args[1];
+        //System.out.println("out: " + outFilePath);
+        try {
+            EmployeeManager employeeManager = new EmployeeManager();
+            List<String> commandStringList = employeeManager.loadCommandStringListFromFile(inFilePath);
+            //System.out.println("cmdStrList: " + commandStringList);
+            List<ICommand> commandList = employeeManager.parseCommandList(commandStringList);
+            //System.out.println("cmdList: " + commandList);
+            String executionResult = employeeManager.executeCommandList(commandList);
+            //System.out.println("exeResult: " + executionResult);
+            employeeManager.saveExecutionResultToFile(outFilePath, executionResult);
+        }catch(Exception e){
+            System.out.println("Error occurred!!!");
```

Comment:
시스템 에러이다 보니 콘솔 에러로 출력하는 것은 어떨까요?

```suggestion
            System.err.println("Error occurred!!!");
```